tests/contrib/forecast/test_util.py

```diff
@@ -90,8 +91,7 @@ def test_reshape_batch(Dist, batch_shape, duration, dim):
 
     actual = reshape_batch(d, batch_shape + (1,))
     assert type(actual) is type(d)
-    if batch_shape:
-        assert actual.batch_shape == batch_shape + (1,)
-    else:
-        assert actual.batch_shape in ((), batch_shape + (1,))
```

Comment:
It seems to me that only LinearHMM is intended to have `actual.batch_shape == ()`. Is there a reason behind?